dds/DCPS/QOS_XML_Handler/optional_ace_xml_utils.mpb

```diff
@@ -1,2 +1,2 @@
-feature(dds4ccm_opendds): ace_xml_utils {
+feature(dcps_xml_handler): ace_xml_utils {
```

Comment:
@jwillemsen where is dcps_xml_handler used as an MPC feature?  Should this mpb file just be removed?